### PR TITLE
fix doc description in app-designer-overview

### PR DESCRIPTION
**Pull request by Chinex-Boroja**
#### What is included in this PR?

1. I made line-edits in `app-designer-overview.rst` which is for https://github.com/odk-x/tool-suite-X/issues/207
2. Updated to documentation for more clarity and easy understanding of detail-views and list-views

### CODE REVIEW

**Comment by Chinex-Boroja** [comment]
Hi @wbrunette @elmps2018 you take a look at the new changes and review. Thank you

**Review by wbrunette — CHANGES_REQUESTED** [review]
I just noticed the way it reads is slightly off as it could be interpreted as list and detailed views are also in figure.  Perhaps a better solution is to simply remove graph from the list and just have list views and detail views.

**Comment by Chinex-Boroja** [comment]
> I just noticed the way it reads is slightly off as it could be interpreted as list and detailed views are also in figure. Perhaps a better solution is to simply remove graph from the list and just have list views and detail views.

Hi @wbrunette, I just made a fix, you can kindly review. Thank you

File: src/app-designer-overview.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ The ODK-X Application Designer is accessed through a :program:`Chrome` browser.
 
 This screen has 6 tabs:
 
-  - :ref:`Preview <app-designer-overview-preview>` (shown above) - used to preview `ODK-X Survey <https://docs.odk-x.org/survey-using/>`_ forms and ODK-X Tables list-, detail- and graph- views (future). Displays these within a user-selected device geometry.
+  - :ref:`Preview <app-designer-overview-preview>` (shown above) - used to preview `ODK-X Survey <https://docs.odk-x.org/survey-using/>`_ forms and ODK-X Tables list-, detail- and graph- views (in future). Displays these within a user-selected device geometry.
```
[the remark this answers is not in the document]
Comment (wbrunette) [reply src/app-designer-overview.rst]: Just graph views got dropped (needed to be rewritten) ... detail, list, and map views make up the essence of tables.

[the remark this answers is not in the document]
Comment (Chinex-Boroja) [reply src/app-designer-overview.rst]: @elmps2018 I have made the fix. Can you kindly review?

[the remark this answers is not in the document]
Comment (elmps2018) [reply src/app-designer-overview.rst]: @Chinex-Boroja the text looks good to me, but @wbrunette are the other types in the future?

File: src/app-designer-overview.rst [hunk 2]
```diff
@@ -12,7 +12,7 @@ The ODK-X Application Designer is accessed through a :program:`Chrome` browser.
 
 This screen has 6 tabs:
 
-  - :ref:`Preview <app-designer-overview-preview>` (shown above) - used to preview `ODK-X Survey <https://docs.odk-x.org/survey-using/>`_ forms and ODK-X Tables list-, detail- and graph- views (future). Displays these within a user-selected device geometry.
+  - :ref:`Preview <app-designer-overview-preview>` (shown above) - used to preview `ODK-X Survey <https://docs.odk-x.org/survey-using/>`_ forms and ODK-X Tables list-views and detail-views (in the future). Displays these within a user-selected device geometry.
```
Comment (wbrunette) [inline src/app-designer-overview.rst:15]: "In the future" is not needed as graph views was the only one in future and was confusing. List and Detail views are both available and are NOT in the future.

**Review by wbrunette — CHANGES_REQUESTED** [review]

**Review by wbrunette — APPROVED** [review]

**Pull request merged**